platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateGenerator.java

```diff
@@ -77,10 +77,7 @@ protected void registerMbean() {
         }
       } catch (Exception e) {
         // objectName is not always non-null because new ObjectName(...) can throw an exception
```

Comment:
Can't ojbectName be null here?

Reply:
objectName will always be non-null if mBeanServer is also non-null. I removed the comment